internal/cmd/argus/credentials/create/create.go

```diff
@@ -18,15 +18,13 @@ import (
 )
 
 const (
-	instanceIdFlag   = "instance-id"
-	hidePasswordFlag = "hide-password"
+	instanceIdFlag = "instance-id"
 )
 
 type inputModel struct {
 	*globalflags.GlobalFlagModel
 
-	HidePassword bool
-	InstanceId   string
+	InstanceId string
```

Comment:
Why remove the `hide-password` functionality?

Reply:
In that sense, I also added documentation to the help of the command stating this